manifests/server.pp

```diff
@@ -334,6 +337,7 @@
   Array[String] $cipher_suites = $::puppet::server_cipher_suites,
   Optional[String] $config_version = $::puppet::server_config_version,
   Integer[0] $connect_timeout = $::puppet::server_connect_timeout,
+  Integer[0] $server_web_idle_timeout = $puppet::params::server_web_idle_timeout,
```

Comment:
In general we strip the ```server_``` prefix in this class. I think the same can be done here and just by named ```web_idle_timeout```. Note that in init.pp it should stay prefixed.